packages/react-resizable-panels/src/PanelResizeHandle.test.tsx

```diff
@@ -72,6 +82,24 @@ describe("PanelResizeHandle", () => {
     expect(element.title).toBe("bar");
   });
 
+  it("resets the global cursor style on unmount", () => {
+    act(() => {
+      root.render(
+        <PanelGroup direction="horizontal">
+          <Panel />
+          <PanelResizeHandle />
+          <Panel />
+        </PanelGroup>
+      );
+    });
+
+    act(() => {
```

Comment:
This test could give a false positive, as it's currently written. You can fix this by verifying that the mock function hasn't been called before you expected.
```suggestion
    });

    expect(cursorUtils.resetGlobalCursorStyle).not.toHaveBeenCalled();

    act(() => {
```

Reply:
Good shout, I've made that change.